string object_name 
float64[] object_pose_array
string object_model_name
---
bool success
